auth/config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  secret: "A_SECRET_KEY",
```

Comment:
This is not good enough for jwt. Read on how to generate RSA public and private key to use for signing the Jwt.